packages/kaizenplatform/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "@userdive/agent": "^1.2.0",
+    "@userdive/provider": "^1.2.0",
```

Comment:
missing private module dependency. It needs for lerna to calculate topological sorting order.